examples/libs/tiny_ttf/lv_example_tiny_ttf_3.c

```diff
@@ -1,3 +1,4 @@
+#include "../../../src/others/observer/lv_observer_private.h"
```

Comment:
The private header files shouldn't be required to "just use" a features. In this file on the basic observer features are used, so `lv_observer_private.h` shouldn't be needed.